cmd/kubelet/app/server.go

```diff
@@ -1260,6 +1260,10 @@ func RunKubelet(ctx context.Context, kubeServer *options.KubeletServer, kubeDeps
 
 	// process pods and exit.
 	if runOnce {
+		if !utilfeature.DefaultFeatureGate.Enabled(features.LegacyNodeRunOnceMode) {
+			panic("Cannot run runonce mode when LegacyNodeRunOnceMode feature gate is disabled")
+		}
+		klog.InfoS("WARNING: RunOnce mode has been deprecated, and will be removed in a future release")
```

Comment:
@thockin I see akin to https://github.com/kubernetes/kubernetes/pull/126084. So, we should drop the language around removal, right?

Reply:
@HirazawaUi Also, we should adjust the language to encourage migrating to alternatives covered in the KEP for this.

Reply:
> Also, we should adjust the language to encourage migrating to alternatives covered in the KEP for this.

Added. Do you think we should add the URL for podman kube? (I believe it's not necessary because the URL for podman kube might change).